indico/modules/events/abstracts/templates/abstract/judge.html

```diff
@@ -40,52 +40,56 @@
-                            {% trans %}Review summary{% endtrans %}
-                        </strong>
+                        {{ track.title }}
                     </div>
-                    <div class="abstract-comment-badges">
-                        {% trans state=_render_abstract_reviewing_state(abstract) -%}
-                            Reviews are {{ state }}
-                        {%- endtrans %}
-                        {% if abstract.score %}
-                            {% set score = abstract.score|round(1) %}
-                            {# TODO: show max score #}
-                            <span class="i-tag highlight" title="{% trans %}Score: {{ score }}/...{% endtrans %}">
+                    <div>
+                        {{ _render_abstract_track_reviewing_state(abstract, track) }}
+                        {% set score = abstract.get_track_score(track) %}
+                        {% if score is not none %}
+                            {% set score = score|round(1) %}
+                            <span class="i-tag highlight outline" title="{% trans %}Score: {{ score }}/...{% endtrans %}">
                                 {{ score }}
                             </span>
                         {% endif %}
                     </div>
                 </div>
-                {% for track in abstract.reviewed_for_tracks|sort(attribute='title') %}
-                    <div class="track-review flex">
-                        <div class="stretch">
-                            {{ track.title }}
-                        </div>
-                        <div>
-                            {{ _render_abstract_track_reviewing_state(abstract, track) }}
-                            {% set score = abstract.get_track_score(track) %}
-                            {% if score is not none %}
-                                {% set score = score|round(1) %}
-                                <span class="i-tag highlight outline" title="{% trans %}Score: {{ score }}/...{% endtrans %}">
-                                    {{ score }}
-                                </span>
-                            {% endif %}
-                        </div>
-                    </div>
-                {% endfor %}
-            </div>
-            <div class="i-box-footer">
-                {{ simple_form(form, fields=form._order, back_button=false, skip_labels=true, submit=_("Judge"),
-                               disabled_until_change=false, form_header_kwargs={'orientation': 'vertical'}) }}
-            </div>
+            {% endfor %}
         </div>
+        <div class="i-box-footer">
+            {{ form_header(form, orientation='vertical') }}
+            {{ form_rows(form, fields=form._order) }}
+            {% call form_footer(form, skip_label=skip_labels) %}
+                <button class="i-button big highlight judge-button"
+                        data-href="{{ url_for('.judge_abstract', abstract, management=management) }}">
```

Comment:
Wouldn't it be cleaner to put this in the `action` of the form and then bind a submit event for the form?